database/index.js

```diff
@@ -17,4 +17,61 @@ const pool = new Pool({
   database: 'fetsy',
 });
 
+const model = {
+  getAll: (cb) => {
```

Comment:
Curious if a getAll is effective when the database grows to 10 mil

Reply:
That's an important point to bring up! The main query to render data on the FE only uses 'getOne', but the legacy code came with an endpoint to 'getAll' so I just kept it for now with my refactor.